src/builder.rs

```diff
@@ -184,6 +184,22 @@ impl<W: Write> Builder<W> {
         self.append(&header, data)
     }
 
+    /// Add a file entry to the archive.
+    ///
+    /// This function will create a header for a file of correct size, mode, and checksum.
```

Comment:
This document is not as detailed as the other methods, but I don't know how to make it.